bindings/python/iota_sdk/client/_node_core_api.py

```diff
@@ -84,18 +84,19 @@ def get_info(self, url: str, auth=None) -> InfoResponse:
 
     # Accounts routes.
 
-    def get_account_congestion(self, account_id: HexStr) -> CongestionResponse:
+    def get_account_congestion(self, account_id: HexStr, work_score: Optional[int]) -> CongestionResponse:
```

Comment:
We should have a default None value, same for the other methods with optional params below
```suggestion
    def get_account_congestion(self, account_id: HexStr, work_score: Optional[int] = None) -> CongestionResponse:
```

Reply:
yup